datasources/northernaustralia/shared/infrastructure.js

```diff
@@ -4,23 +4,32 @@ const getFromCatalogPath = require("../../getFromCatalogPath");
 
 const externalCatalogs = require("../shared/externalCatalogs");
 
-function getElectricity(aremi) {
+function getElectricity() {
   const electricity = Object.assign(
     {},
-    getFromCatalogPath(aremi, ["Electricity Infrastructure", "Transmission"]),
+    getFromCatalogPath(externalCatalogs.aremi, [
+      "Electricity Infrastructure",
+      "Transmission"
+    ]),
     { name: "Electricity" }
   );
-  electricity.items = electricity.items.filter(
-    item => item.name !== "Western Australia"
-  );
+
+  electricity.items = [
+    ...electricity.items.filter(item => item.name !== "Western Australia"),
+    getFromCatalogPath(externalCatalogs.nationalmap, [
+      "National Datasets",
+      "Utility",
+      "Powerlines"
```

Comment:
The Powerlines layer is out of date compared to the Transmission Lines - since the Trans Lines are being added this one probably shouldn't be.